packages/data-stores/src/queries/use-wpcom-site.ts

```diff
@@ -9,6 +9,7 @@ export function useWpcomSite( siteId: number | string | undefined, enabled = tru
 			wpcomRequest< SiteDetails >( {
 				path: '/sites/' + encodeURIComponent( siteId as string ),
 				apiVersion: '1.1',
+				query: '?force=wpcom',
```

Comment:
```suggestion
				query: 'force=wpcom',
```
Otherwise, the URL looks like `https://public-api.wordpress.com/rest/v1.1/sites/XXXXX?http_envelope=1&?force=wpcom` (notice the `&?force`)